.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
It's important to ensure that `db.sqlite3` is included in the `.gitignore` file to prevent the database file from being committed to the repository. This is crucial for maintaining data privacy and avoiding unnecessary data conflicts.